src/main/java/me/zeroeightsix/fiber/annotation/Settings.java

```diff
@@ -3,14 +3,36 @@
 import me.zeroeightsix.fiber.annotation.convention.NoNamingConvention;
 import me.zeroeightsix.fiber.annotation.convention.SettingNamingConvention;
 
+import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 
+/**
+ * Indicates a type represents a structure in a configuration file.
```

Comment:
What indicates this?

Reply:
The annotation indicates this. I'm not sure how I'd word this differently. Thoughts?

Reply:
I meant because there are two annotations. And I feel like it may be unclear, although I haven't used to code past reading it so it may be very clear which of the two it is when you have used it more.

Reply:
> Two annotations
`@Setting` and `@Settings`? If you are referring to those two, I'd opt for leaving the javadoc the same. Both annotations have different `@Target`s, so using them wrongly would result in compiler errors.